config/jobs/kubernetes/sig-arch/conformance-gate.yaml

```diff
@@ -0,0 +1,18 @@
+periodics:
+- interval: 4h
+  name: conformance-gate
+  annotations:
+    testgrid-dashboards: conformance-apisnoop
+    testgrid-tab-name: conformance-gate
+    test-grid-alert-email: en-server-prow-test-alert@google.com
+    testgrid-num-failures-to-alert: '1'
+  decorate: true
+  spec:
+    containers:
+    - name: apisnoop-gate
+      image: gcr.io/k8s-staging-apisnoop/snoopdb:v20200923-v2020.09.23-1-gb3afc41
+      ports:
+      - name: postgres
+        containerPort: 5432
```

Comment:
Can this be dropped?

Reply:
ah, yep!  This has been dropped, @spiffxp.